_includes/pbc/all/install-features/202407.0/install-the-cart-feature.md

```diff
@@ -199,7 +223,67 @@ class RouterDependencyProvider extends SprykerRouterDependencyProvider
 
 {% endinfo_block %}
 
-### 4) Set up widgets
+### 4) Set up behavior
+
+Activate the following plugins:
+
+| PLUGIN                               | SPECIFICATION                                                    | PREREQUISITES | NAMESPACE                                 |
+|--------------------------------------|------------------------------------------------------------------|---------------|-------------------------------------------|
+| CartBlockMiniCartViewExpanderPlugin  | Expands the provided `MiniCartView.content` with a mini cart view. |           | SprykerShop\Yves\CartPage\Plugin\CartPage |
+
+**src/Pyz/Yves/Router/RouterDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\CartPage;
+
+use SprykerShop\Yves\CartPage\CartPageDependencyProvider as SprykerCartPageDependencyProvider;
+use SprykerShop\Yves\CartPage\Plugin\CartPage\CartBlockMiniCartViewExpanderPlugin;
+
+class CartPageDependencyProvider extends SprykerCartPageDependencyProvider
+{
+    /**
+     * @return array<\SprykerShop\Yves\CartPageExtension\Dependency\Plugin\MiniCartViewExpanderPluginInterface>
+     */
+    protected function getMiniCartViewExpanderPlugins(): array
+    {
+        return [
+            new CartBlockMiniCartViewExpanderPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+It's an optional step. Only B2C Demo Shop uses this plugin due to custom mini cart implementation.
```

Comment:
This doesn't look like verification

Reply:
you right, do you have any suggestion where I could left such information?